README.md

```diff
@@ -22,19 +22,19 @@ As soon as you keep a reference to an object, array, arrow function, ... you do
 
 ```js
 import myLib from 'my-lib'
-import { iterate } from 'leakage'
+import { iteration } from 'leakage'
```

Comment:
Thanks for fixing this! It took me a moment to figure out why my import wasn't working before I discovered the name was different.